### PR TITLE
Handle undefined `target` in SelectWidget's onBlur and onFocus

### PR DESCRIPTION
**Pull request by huyph**
### Reasons for making this change

[Please describe them here]
When using the default SelectWidget from @rjsf/semantic-ui@5.5.0, I observed these errors from `SelectWidget.tsx` 

```
Cannot read properties of undefined (reading 'value')
TypeError: Cannot read properties of undefined (reading 'value')
    at Object._onBlur [as **onBlur**] ...
...
```

```
Cannot read properties of undefined (reading 'value')
TypeError: Cannot read properties of undefined (reading 'value')
    at Object._onBlur [as **onFocus**] ...
...
```

### Checklist

- [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://rjsf-team.github.io/react-jsonschema-form/docs/contributing) of the Markdown text I've added
- [x] **I'm adding or updating code**
  - [x] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [ ] I've updated [docs](https://rjsf-team.github.io/react-jsonschema-form/docs) if needed
  - [x] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/main/CHANGELOG.md) with a description of the PR
- [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/semantic-ui/src/SelectWidget/SelectWidget.tsx [hunk 1]
```diff
@@ -80,10 +80,10 @@ export default function SelectWidget<T = any, S extends StrictRJSFSchema = RJSFS
...
     onChange(enumOptionsValueForIndex<S>(value as string[], enumOptions, optEmptyVal));
   // eslint-disable-next-line no-shadow
-  const _onBlur = (_: FocusEvent<HTMLElement>, { target: { value } }: DropdownProps) =>
-    onBlur(id, enumOptionsValueForIndex<S>(value, enumOptions, optEmptyVal));
-  const _onFocus = (_: FocusEvent<HTMLElement>, { target: { value } }: DropdownProps) =>
-    onFocus(id, enumOptionsValueForIndex<S>(value, enumOptions, optEmptyVal));
+  const _onBlur = (_: FocusEvent<HTMLElement>, { target }: DropdownProps) =>
+    onBlur(id, enumOptionsValueForIndex<S>(target && target.value, enumOptions, optEmptyVal));
+  const _onFocus = (_: FocusEvent<HTMLElement>, { target }: DropdownProps) =>
+    onFocus(id, enumOptionsValueForIndex<S>(target && target.value, enumOptions, optEmptyVal));
```
Comment (heath-freenome) [inline packages/semantic-ui/src/SelectWidget/SelectWidget.tsx:86]: Is this the only component that has this issue? Maybe the same change needs to be made for all of the other widgets that deal with onBlur/onFocus?

**Comment by heath-freenome** [comment]
@huyph I wondering when you'll get a change to address my questions.  Also you'll have to resolve conflicts now.

**Comment by hfjohn123** [comment]
Hi I am still having this problem with the newest version when I am utilizing semantic UI Theme. Is there any workaround?

**Comment by heath-freenome** [comment]
@hfjohn123 I'm not sure why @huyph closed this PR. If you want to pick up this fix and address the additional potential changes I encourage you to do so and open your own PR.

File: packages/semantic-ui/src/SelectWidget/SelectWidget.tsx (hunk 1 above)
Comment (huyph) [reply packages/semantic-ui/src/SelectWidget/SelectWidget.tsx:86]: I will look into it.

**Comment by huyph** [comment]
Hi @heath-freenome, I accidentally closed this PR. Sorry for the confusion.

I will try to get back to it within the next few weeks. Feel free to collaborate with me on this PR @hfjohn123

**Comment by ahmedsakri** [comment]
Hi, any update on this P.R? as i am too facing this issue. Thanks!

**Comment by heath-freenome** [comment]
@ahmedsakri Maybe you'll need to collaborate with @huyph to get this fixed?

**Comment by heath-freenome** [comment]
@huyph Seems like this stalled?

**Comment by heath-freenome** [comment]
Closing in favor of https://github.com/rjsf-team/react-jsonschema-form/pull/4227

**Pull request closed without merging**